static/app/types/integrations.tsx

```diff
@@ -490,8 +490,10 @@ export type CodeownersFile = {
 
 export type FilesByRepository = {
   [repoName: string]: {
-    authors?: {[email: string]: CommitAuthor};
-    types?: Set<string>;
+    [fileName: string]: {
```

Comment:
This type def was actually broken, see getFilesByRepository in  static/app/views/releases/detail/utils.tsx